unit-tests/live/safety_preset/test-get-set-xu-cmd.py

```diff
@@ -0,0 +1,62 @@
+# License: Apache 2.0. See LICENSE file in root directory.
+# Copyright(c) 2022 Intel Corporation. All Rights Reserved.
+
+# test:device S585
+import pyrealsense2 as rs
+from rspy import devices, log, test, file, repo
+
+#############################################################################################
+# Tests
+#############################################################################################
+
+test.start("Init")
+try:
+    ctx = rs.context()
+    dev = ctx.query_devices()[0]
+    safety_sensor = dev.first_safety_sensor()
+except:
+    test.unexpected_exception()
+test.finish()
+
+#############################################################################################
+
+test.start("Valid Get/Set")
+try:
+    if safety_sensor.supports(rs.option.active_safety_preset_index):
```

Comment:
I think we want
test.check_equal(safety_sensor.supports(rs.option.active_safety_preset_index))
or ignore it at all.
If it's S585, it should support, if it's not it an issue.
Here if it's not the test will pass and noting will be checked.

Reply:
did not understand you...

Reply:
You check if this option is supported, then you check all the logic, if it's not supported the test pass and nothing is checked
I say:.
On S585 if this option is not supported it's a failure.
We should test all without this condition, it will only run on S585 device that has your code inside.
Sounds right?

Reply:
in my test, if this if return false, it will raise exception, so I think it will behave correctly, and it also checks that we are registering this option in librealsense by invoking the support method.

Reply:
done